File: project_server/java_server/src/main/java/com/example/blog/Model/Article.java
package com.example.blog.Model;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.util.Date;
import java.util.List;

@Data
@TableName("articles")
public class Article {

    private Integer id;

    private String title;

    private String content;

    private Date createTime;

    private Date updateTime;

    private int status;

    private int comments_sum;

    private int view_sum;
}

File: project_server/go_server/model/article.go
package model

import "time"

// Articles represents the articles table in the database
type Articles struct {
	ID          uint       `gorm:"primaryKey;autoIncrement"`                              // 主键，自增
	Title       string     `gorm:"type:varchar(255);not null;collate:utf8mb4_general_ci"` // 文章标题，非空
	Content     string     `gorm:"type:text;not null;collate:utf8mb4_general_ci"`         // 文章内容，非空
	CreateTime  time.Time  `gorm:"type:datetime;not null"`                                // 创建时间，非空
	UpdateTime  *time.Time `gorm:"type:datetime"`                                         // 更新时间，允许为空
	Status      int        `gorm:"type:tinyint;default:1"`                                // 文章状态，默认为1
	CommentsSum int        `gorm:"type:int;default:0"`                                    // 评论数量，默认为0
	ViewSum     int        `gorm:"type:int;default:0"`                                    // 浏览数量，默认为0
}
